src/OmniSharp.MSBuild/ProjectLoader.cs

```diff
@@ -67,11 +67,12 @@ public ProjectLoader(MSBuildOptions options, string solutionDirectory, Immutable
             return globalProperties;
         }
 
-        public (MSB.Execution.ProjectInstance projectInstance, ImmutableArray<MSBuildDiagnostic> diagnostics) BuildProject(string filePath)
+        public (MSB.Execution.ProjectInstance projectInstance, ImmutableArray<MSBuildDiagnostic> diagnostics) BuildProject(
+            string filePath, Dictionary<string, string> configurationsInSolution)
         {
```

Comment:
Use `IReadOnlyDictionary<string, string>` or `IEnumerable<KeyValuePair<string, string>>`. Or maybe `ImmutableDictionary<string, string>` to make it consistent with current API.